app/serializers/restaurant_serializer.rb

```diff
@@ -0,0 +1,4 @@
+# Restaurant serializer
+class RestaurantSerializer < ApplicationSerializer
+  attributes :id, :name, :cuisine_id, :rating, :ten_bis, :address
```

Comment:
don't you get it from the application serilizer??

Reply:
exactly, you don't need the id here.